jakarta.batch.arquillian.exec-parent/README.md

```diff
@@ -33,11 +33,17 @@ Create a new maven project that:
 
 * Uses this maven artifact as the parent
 * Contains the Arquillian container for the target runtime, including all its dependencies and configuration for it
-* (Optionally but strongly recommended) Contains the `maven-dependency-plugin` in the list of plugins. This plugin 
+* (Required) Contains the `maven-dependency-plugin` in the list of plugins. This plugin 
```

Comment:
As mentioned in Slack, I think we have to take another pass through and make sure the TCK Reference Guide clearly defines the "official" list of Required vs Optional config..and either delete or cross-reference these .md files.  I like having them since the full guide is a large document to read through.

Reply:
But the guide could be updated in a (final) follow-up PR.